cbv3_django_prototype/cbv3_django_prototype/resources/models.py

```diff
@@ -65,11 +66,9 @@ class Resource(models.Model):
 
     paid = models.BooleanField(null=True)
 
-    # JSONB for a simplified DB Schema and prototype for now.
-    #tags = pg.JSONField()
     # Allow tags to be used across entities
     # E.g. so we can create composite views showing all entities sharing a common tag
-    tags = TaggableManager()
+    tags = TaggableManager(blank=True)
```

Comment:
I'm not sure if this is intentional, but the resulting behaviour still requires the `tags` property to be defined as an empty array and so it is a required field when submitting resources.

Reply:
That looks to be a merge oversight on my part.  I will fix this in a push to this PR.  I am going with option number 2:  `tags = TagggableManager(blank=True)`, to see if it removes the tags being required.  That was the intended behavior.

Reply:
Belay that.  I can't make the field nullable.  I will log as a separate issue, but it looks like `taggit` needs **something** there, and that makes the field required.  We might have to put code in to insert the value if it is missing from the request.

Reply:
Logged issue #74 for this.